src/bandersnatch_filter_plugins/latest_name.py

```diff
@@ -32,6 +34,17 @@ def initialize_plugin(self) -> None:
             return
         if self.keep > 0:
             logger.info(f"Initialized latest releases plugin with keep={self.keep}")
+        try:
+            sort_by = self.configuration["latest_release"]["sort_by"]
+            if sort_by in ["time", "version"]:
+                self.sort_by = sort_by
```

Comment:
We should probably at least log.debug if someone sets a bad config option + state we're sticking with default of version sorting

Reply:
added a debug log warning